neqo-crypto/src/lib.rs

```diff
@@ -103,7 +103,9 @@ fn version_check() -> Res<()> {
 /// This allows us to use SSLTRACE in all of our unit tests and programs.
 #[cfg(debug_assertions)]
 fn enable_ssl_trace() -> Res<()> {
-    let opt = Opt::Locking.as_int();
+    use ssl::PRInt32;
+
+    let opt = Opt::Locking as PRInt32;
```

Comment:
Not a strong opinion. It was nice that the potentially unsafe cast (i.e. `as`) was previously closer to the implementation (i.e. `Opt`).